perfmetrics/scripts/presubmit/print_results.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+# For fetching fio_metrics from fio
+sys.path.append("./perfmetrics/scripts/")
```

Comment:
how is fio_metrics getting gsheet reference without this line
https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/fio/fio_metrics.py#L19

Reply:
we are not using gsheet in our python script.
We are using only [get_metrics](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/fio/fio_metrics.py#L437) function which is returning list from json file.